.github/workflows/release-rai-test-utils.yml

```diff
@@ -26,31 +26,55 @@ jobs:
         with:
           python-version: 3.8
 
+      - uses: conda-incubator/setup-miniconda@v2
+        with:
+          auto-update-conda: true
+          python-version: ${{ matrix.pythonVersion }}
+
+      - if: ${{ matrix.operatingSystem != 'macos-latest' }}
+        name: Install pytorch on non-MacOS
+        shell: bash -l {0}
+        run: |
+          conda install --yes --quiet pytorch torchvision captum cpuonly -c pytorch
+
+      - if: ${{ matrix.operatingSystem == 'macos-latest' }}
+        name: Install Anaconda packages on MacOS, which should not include cpuonly according to official docs
+        shell: bash -l {0}
+        run: |
+          conda install --yes --quiet pytorch torchvision captum -c pytorch
+
       - name: update and upgrade pip, setuptools, wheel, and twine
+        shell: bash -l {0}
```

Comment:
Why do you need bash -1 {0} in all the workflow steps?

Reply:
As per @imatiach-msft's suggestion and prior testing that this bash command will be needed in places where both python & conda (because of torch) are setup. SImilar to `CI-Python.yml`.